sync/syncer.go

```diff
@@ -496,7 +496,8 @@ func (s *Syncer) syncAtxs(atxIds []types.AtxId) ([]*types.ActivationTx, error) {
 					s.Warning("atx %v not valid %v", atx.ShortId(), err)
 					continue
 				}
-				unprocessedAtxs[atx.Id()] = &atx
+				tmp := atx
```

Comment:
this is fine for now, ill add a task to try and consolidate the txs and atxs code to avoid duplication and bugs